apps/mobile/src/features/Settings/useCases/changeAppSettings/ApplicationSettingsScreen.tsx

```diff
@@ -131,28 +221,26 @@ const CrashReportsSwitch = ({
 }: {
   crashReportEnabled: boolean
 }) => {
-  const {enable, disable} = useCrashReports()
-  const [isLocalEnabled, setIsLocalEnabled] = React.useState(crashReportEnabled)
+  const {enable, disable, enabled} = useCrashReports()
+  const [isLocalCrashReportEnabled, setIsLocalCrashReportEnabled] =
```

Comment:
this can be consumed with useSyncStorageToState, then the React state can be dropped